ui/src/app/components/workspace/workspace.component.html

```diff
@@ -15,13 +15,15 @@
     </div>
   </div>
   <div class="container">
-    <h2>Configure schema</h2>
     <div class="summary">
```

Comment:
This text should be aligned to be vertically centre: https://drive.google.com/file/d/1KypG7f99XnXSTrrjkju1yX-_zx2h3AQu/view?usp=share_link&resourcekey=0-0AFTriAGz4wgWsYtkdiGbA

Reply:
Updated

Reply:
Updated